Makefile

```diff
@@ -39,7 +39,7 @@ deploy: manifests
 	kustomize build config/default | kubectl apply -f -
 
 # Deploy controller dev image in the configured Kubernetes cluster in ~/.kube/config
-dev-deploy: manifests
```

Comment:
Why was this omitted?

Reply:
Manifest gen requires libgit2 to be installed which wont work in the CI as this is run outside of the CI container. That is why it had to be removed.